src/sort.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Sergiors\Functional;
+
+/**
+ * @author Sérgio Rafael Siqueira <sergio@inbep.com.br>
+ *
+ * @return mixed
+ */
+function sort(/* ...$args */)
+{
+    $args = func_get_args();
+
+    $sort = function (array $xs, callable $fn) {
```

Comment:
I believe the function must come before. Otherwise, there is no sense in `partial`. Taking first the function would behave like `map`. Thus, you can write a function that sorts several lists based in a function.